go.mod

```diff
@@ -25,7 +25,7 @@ require (
 	github.com/google/renameio v0.1.0
 	github.com/imdario/mergo v0.3.13
 	github.com/opencontainers/go-digest v1.0.0
-	github.com/openshift/api v0.0.0-20220525145417-ee5b62754c68
+	github.com/openshift/api v0.0.0-20220824134416-0f86a223d4bc
```

Comment:
This deprecates a bunch of fields I think, which is why verify is failing. I think you would need to update the fields (e.g. `APIServerInternalIP` -> `APIServerInternalIPs`), note the plural

Reply:
Hi @creydr @cybertron These fields were introduced in https://github.com/openshift/api/pull/1152 and your enhancement https://github.com/openshift/enhancements/pull/1048 mentions "We will need to wire in the new VIPs to the manifests and configuration templates." Is anyone working on this MCO change to handle multiple VIPs yet, or is there a simple way to proceed with this API bump in the mean time?

Reply:
> This deprecates a bunch of fields I think, which is why verify is failing. I think you would need to update the fields (e.g. `APIServerInternalIP` -> `APIServerInternalIPs`), note the plural

@yuqi-zhang I just updated the PR to use `APIServerInternalIPs` instead of `APIServerInternalIP`. But I am not familiar with that part of the code and it looks like my changes could be vulnerable to _slice out of bounds_ types failure.

Reply:
Yes, I agree we should probably wait for input from networking team to make sure

Reply:
Also a lot of yaml files refer to `apiServerInternalIP`

Reply:
Sorry not really familiar with this part of MCO. If anyone from MCO wants to take over this PR to bump the openshift/api, it would be really helpful.

Reply:
@yuqi-zhang I have removed the commit https://github.com/openshift/machine-config-operator/commit/28e3268f70b2023f60e3570850c4bdca1992daa6 from this PR since @cybertron is working on fixing it https://github.com/openshift/machine-config-operator/pull/3308#issuecomment-1231887906